jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -205,6 +205,19 @@ var module = module;
         // if the macaroon is valid.
         false
       );
+    },
+
+    /**
+      Lists all the templates for cloud credentials for a specific user.
+
+      @method listTemplates
+      @param callback {Function} A callback to handle errors or accept the data
+          from the request. Must accept an error message or null as its first
+          parameter and the response data as its second.
+    */
+    listTemplates: function(callback) {
+      var url = this.jemURL + '/template';
+      return _makeRequest(this.bakery, url, 'GET', null, callback, true);
```

Comment:
Yeah, most other API functions have some sort of translation that happens. This was a case where I put in the translation layer and then kept simplifying until it was eventually non-existent. That said, I think I went too far and missed the data.templates bit that @jaycee pointed out. I'll add a minimal translation layer back in.